### PR TITLE
添加当天作业过滤

### PR DESCRIPTION
**Pull request by SunFlowerY1**
目前kylin支持的最小 job timeFilter 粒度为 'last one day'，我们在生产环境中,通常是凌晨开始调度任务,执行基于前一天数据的 cube 构建任务，添加当天作业过滤，为我们监控、维护当天任务执行情况带来了便利

### CODE REVIEW

**Comment by nichunen** [comment]
@SunFlowerY1 Hi, please commit and submit pr with information in English.  BTW, would you please open a jira to describe your enhancement?

**Comment by SunFlowerY1** [comment]
> @SunFlowerY1 Hi, please commit and submit pr with information in English. BTW, would you please open a jira to describe your enhancement?

OK

File: webapp/app/js/model/jobListModel.js [hunk 1]
```diff
@@ -25,15 +25,15 @@ KylinApp.service('JobList',function(JobService,$q){
     this.jobs={};
     this.jobFilter = {
         cubeName : null,
-        timeFilterId : 1,
+        timeFilterId : 0,
```
Comment (nichunen) [inline webapp/app/js/model/jobListModel.js]: Hi, it's more reasonable to keep the default setting as "last one week".

Comment (SunFlowerY1) [reply webapp/app/js/model/jobListModel.js]: Hi, i hava kept the default setting as "last one week" and make it possible to get the job time filter default value from kylin configuration file, thank you very much!

**Pull request merged**